src/models/game.ts

```diff
@@ -5,7 +5,7 @@ export interface IGame extends Document {
   _id: mongoose.Types.ObjectId;
   black: ITeam;
   white: ITeam;
-  moves: string[];
+  moves: string;
```

Comment:
moves is just a string now instead of an array?
Does this just refer to a single instance of the algebraic notation e.g "Ke4"?